modules/test/playwright/tests/object-web/objectEntries.spec.ts

```diff
@@ -640,3 +643,193 @@ test.describe('Manage object entries through View Object Entries', () => {
 		);
 	});
 });
+
+test.describe('Manage bound object entries with update root permission', () => {
+	test('can add bound object entries with update root permission', async ({
+		apiHelpers,
+		page,
+		viewObjectEntriesPage,
+	}) => {
+		test.setTimeout(180000);
```

Comment:
Hey @G-Sa, did you face some timeout issues when trying to run your test without this property?